Core/Src/obc_handler.c

```diff
@@ -115,9 +120,16 @@ int obc_handle_command(uint8_t obc_cmd) {
 #endif
         return 0;
     }
+    case IRIS_TRANSFER_LOG: {
+        clear_and_dump_buffer();
```

Comment:
This sequence will cause the rest of a page to be wasted if the logger doesn't write the whole page in this dump. Does it handle this situation? 

Also, the first thing transfer_log_to_obc() does is dump the buffer